### PR TITLE
Update messages_nl.properties

### PR DESCRIPTION
**Pull request by Remski01**

### CODE REVIEW

**Review by Xeyame — DISMISSED** [review]

**Comment by Remski01** [comment]
I not sure if it also is pushe into the 1.13 branch, so far i see its currently only in the EssentialsX:2.x Branch, also. i still need to make a few other changes in the file.

However, i currently on vacation and do not have a pc to check message's ingame.

**Comment by Xeyame** [comment]
They will probably merge the 1.13 branch into the main one once it's "done", changes will be merged then

**Comment by Remski01** [comment]
I have an question. 

As i was looking on it i noticed the previous editors decide to translate "Home" not always, as now. You have sometimes "Home" and "Huis", it is maybe not better to let is stay always "Home" is that is the command to use it?

ps: Maybe not yet approve it as i still going make more changes when i'm back from vacation where i can test them, and i mean testing as in if it have any ingame typo's.

**Comment by Xeyame** [comment]
It probably depends on the context. As in a sentice "huis" is used, but when we talk about an command, home is used. However, in for example German, the translated word is always used.

**Comment by Remski01** [comment]
Good point, i'm do the home related message's later.

**Comment by mdcfe** [comment]
@DutchUser Let us know when you're ready to merge this.

**Comment by Remski01** [comment]
Yes, sure. However, its going be next week(8 days), then i'm back from vacation and the i can check a few ingamee to be sure they are correct formatted. also,

i noticed that every locale refer to "config.yml" where it should say "kits.yml" in the line `createKitSuccess=`, its prob because it was first the config and later was placed to Kits.yml, but not get updated in the locale's.

Would you like if i update them to in this PR? Or make an seperated?

**Comment by mdcfe** [comment]
@DutchUser Good spot! It'd be great if it was fixed in a PR, but ideally it should be a separate PR since this PR is specifically for the Dutch locale file.

**Comment by Remski01** [comment]
Sure, i make an seprated PR the for those one's.

**Comment by mdcfe** [comment]
Is *this* PR ready to merge?

**Comment by Remski01** [comment]
No, still working on it, but our server have some... things that need to care of, so i cant finish it atm, or can i make an other PR when i have time again? So that it is atleast more updated

File: Essentials/src/messages_nl.properties [hunk 1]
```diff
@@ -502,8 +502,8 @@ teleporting=\u00a76Bezig met teleporteren...
 teleportInvalidLocation=De waarde van co\u00f6rdinaten kan niet hoger dan 30000000 zijn
 teleportNewPlayerError=\u00a74Fout bij het teleporteren van nieuwe speler\!
 teleportRequest=\u00a7c{0}\u00a76 vraagt of hij naar u kan teleporteren.
-teleportRequestAllCancelled=\u00a76All outstanding teleport requests cancelled.
-teleportRequestSpecificCancelled=\u00a76Outstanding teleport request with {0} cancelled.
+teleportRequestAllCancelled=\u00a76Alle openstaaande teleport verzoeken zijn geannuleerd.
+teleportRequestSpecificCancelled=\u00a76Alle openstaande teleport verzoeken van {0} geannuleerd.
```
Comment (mdcfe) [inline Essentials/src/messages_nl.properties]: I don't think this should contain `Alle`? It's for **one** specific teleport request, unlike the message above which is for all of them. (Correct me if I'm wrong and this actually makes more sense in Dutch.)

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: Essentials/src/messages_nl.properties (hunk 1 above)
Comment (Remski01) [reply Essentials/src/messages_nl.properties]: Uhm.... That is an typo, it should say

``teleportRequestSpecificCancelled=\u00a76Openstaande teleport verzoek van {0} is geannuleerd.``

**Comment by mdcfe** [comment]
@DutchUser If the `teleportRequestSpecificCancelled` typo gets fixed, I can merge it now, or I can leave the PR open until you're ready - I don't mind either way.

File: Essentials/src/messages_nl.properties (hunk 1 above)
Comment (mdcfe) [reply Essentials/src/messages_nl.properties]: @DutchUser Is there any chance you could fix that message? Once done, I can merge this.

**Review by mdcfe — CHANGES_REQUESTED** [review]

**Review by mdcfe — APPROVED** [review]
If this is ready, I'm happy to merge this now.

**Comment by Remski01** [comment]
Its ready.

**Pull request merged**